config/checkstyle/checkstyle.xml

```diff
@@ -271,14 +229,7 @@ page at http://checkstyle.sourceforge.net/config.html -->
 
         -->
 
-        <module name="ModifierOrder">
-            <!-- Warn if modifier order is inconsistent with JLS3 8.1.1, 8.3.1, and
```

Comment:
Bring this comment back. I like their idea of putting comment inside of xml element. Another option is to put comment before module element on the same level. Either way the style should be consistent.